integration-tests/functions/src/api/redirect-me.ts

```diff
@@ -0,0 +1,9 @@
+import { GatsbyAPIFunctionResponse, GatsbyAPIFunctionRequest } from "gatsby"
+
+export default function topLevel(
+  req: GatsbyAPIFunctionRequest,
+  res: GatsbyAPIFunctionResponse
+) {
+  // res.json(req.cookies)
```

Comment:
do you need this line?

Reply:
oh no, thanks, removed. Pushed a fix to a test too